### PR TITLE
Delete legacy parsing/validation DAGs

### PR DESCRIPTION
**Pull request by atvaccaro**
# Description

Progress towards https://github.com/cal-itp/data-infra/issues/2109

Deletes the following
1. Legacy RT parsing/validation DAG and associated image/CI
2. Legacy Schedule downloading/validation DAG and downstream DAGs
3. Legacy RT external tables

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation
- [ ] agencies.yml

## How has this been tested?
Schedule DAGs are currently paused; waiting to pause RT until December is done.

### CODE REVIEW

File: docs/_toc.yml [hunk 1]
```diff
@@ -89,7 +89,6 @@ parts:
       - file: services/overview
         sections:
         - file: services/gtfs-ckan-uploader
-        - file: services/gtfs-rt-archive
```
Comment (lauriemerrell) [inline docs/_toc.yml:92]: Do you need to rebase? I thought you already deleted this in #2110

**Review by lauriemerrell — CHANGES_REQUESTED** [review]
Diff is kind of hard to parse compared with #2110, want to make it clearer which changes are actually new here vs. artifact of branching

**Comment by atvaccaro** [comment]
> Diff is kind of hard to parse compared with #2110, want to make it clearer which changes are actually new here vs. artifact of branching (cited one inline example but I think there are others)

Ah yeah sorry, I forgot to rebase after merging the other PR

**Review by lauriemerrell — APPROVED** [review]
Seems ok to me, only question is whether there's stuff in `sandbox` that we want to delete also? Low priority
